events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
-# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop hosted by Interspecies.IO
+# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop
 
-**TODO:** Copy describing the meeting
-- Time/Date (with link to https://savvytime.com/converter/pst-to-mst-est-cest or some such calculator)
-- less info in the title and more here?
-- Attendees? (greater consideration given the number of people we had in attendance)
+## hosted by Interspecies.IO
```

Comment:
I wonder how this will render. My fear is that it will be a bit kludgy. I went back and forth on this. Let's find out!